### PR TITLE
Modify README and add LICENSE

### PR DESCRIPTION
**Pull request by RobertoCorti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,14 +30,33 @@ To activate the virtual environment then run
...
+streamlit run prototype.py
 ```
-poetry remove <package>
\ No newline at end of file
+This will start the GPTravel app using Streamlit. You can then access the web app through your browser.
+
+## Next Steps 🌟
+Here are some suggested next steps to enhance GPTravel:
+
+* Creation of a Travel Score based on the diversity of the activities generated by GPTravel.
```
Comment (stefano-polo) [inline README.md:46]: @RobertoCorti  I do not understand this point. We have implemented a metric based on entropy information to provide a score to the variety of the travel plan activities. Maybe do you mean something else?

Comment (RobertoCorti) [reply README.md:47]: Yes, maybe I should remove it...

File: README.md [hunk 2]
```diff
@@ -1,7 +1,21 @@
-# gptravel
+# GPTravel ✈️
 ![Tests](https://github.com/RobertoCorti/gptravel/actions/workflows/python-tests.yml/badge.svg)
+![GitHub Tags](https://img.shields.io/github/tag/RobertoCorti/gptravel.svg)
```
Comment (stefano-polo) [inline README.md:4]: @RobertoCorti  I would suggest to add also the License badge if possible. You can check this page: https://gist.github.com/lukas-h/2a5d00690736b4c3a7ba

Comment (RobertoCorti) [reply README.md:4]: Ok, I'll add

**Comment by stefano-polo** [comment]
LGTM!

**Review by stefano-polo — APPROVED** [review]

**Review by stefano-polo — APPROVED** [review]

**Review by stefano-polo — APPROVED** [review]

**Pull request merged**